oauth2client/contrib/gce.py

```diff
@@ -107,7 +117,7 @@ def serialization_data(self):
             'Cannot serialize credentials for GCE service accounts.')
 
     def create_scoped_required(self):
-        return not self.scope
+        return False
```

Comment:
@nathanielmanistaatgoogle Note this has also changed since obviously no scopes are ever required (still won't prevent GCE tokens from not having the right scopes, it's just not something this class can do anything about).